src/components/screen-header/FullScreenHeader.tsx

```diff
@@ -12,8 +18,28 @@ export function FullScreenHeader(props: ScreenHeaderProps) {
   const backgroundColor = getStaticColor(themeName, themeColor).background;
 
   return (
-    <View style={[{backgroundColor}, {paddingTop: top}]}>
+    <View style={{backgroundColor, paddingTop: top}}>
       <ScreenHeader {...props} />
     </View>
   );
 }
+
+export function LargeFullScreenHeader(
+  props: LargeScreenHeaderProps & {
+    buttonProps?: Omit<LargeHeaderButtonProps, 'color'>;
+  },
```

Comment:
I think it is slightly better if all parameters are defined in the `LargeScreenHeaderProps`, especially when it is exported. Now the parameter `buttonProps` is kind of "hidden".